hubmap/frontend/cypress/support/commands.js

```diff
@@ -23,3 +25,22 @@
 //
 // -- This is will overwrite an existing command --
 // Cypress.Commands.overwrite("visit", (originalFn, url, options) => { ... })
+Cypress.Commands.add('stubLoginUser', () => {
+    const log = Cypress.log({
+        name: 'stubLoginUser',
+    })
+    cy.server()
+    .setCookie('email', 'test@gmail.com')
+    .route('http://localhost:8000/auth/login/globus/', 'fixture:login')
+    .then(()=>{log.snapshot().end()})
```

Comment:
Is this necessary? This is short enough, and only used in one place, so I think it's premature to pull it into a utility like this: It only adds another jump when I'm trying to understand the code.

Reply:
@mccalluc This is the recommended stub for url redirects to use this. And please do correct me if I am wrong, I am only a day-old in knowing cypress, we need to add command to add a stub for url redirects outside of domain. cy.route/cy.request do not work for url redirects. Sorry but may be it is not clear what you specifically are looking or meaning, can you please help understand a detail ?

Reply:
If not we will just set cookies in stubs without routes. whichever you suggest.

Reply:
https://github.com/cypress-io/cypress/issues/2621